src/client/index.tsx

```diff
@@ -260,3 +261,19 @@ export const getProviders = GET(async () => {
     headers: { 'cache-control': 'max-age=86400' },
   });
 });
+
+export const getLicenses = GET(async (locationsId: number) => {
+  'use server'
+  const url = new URL(import.meta.env.VITE_API_BASE_URL);
+  url.pathname = `/v3/licenses/${locationsId}`
```

Comment:
There is no `v3/licenses/{locationsId}` so this shouldnt work. Lets make this function do two things to achieve the intended result:

1. fetch the location by id
2. loop through the location's licenses to get the licenses_id 
3. fetch licenses at /v3/licenses
4. filter the licenses to include just those with the relevant licenses_id(s)
5. return the resulting array of licenses